src/api-server.ts

```diff
@@ -467,12 +467,9 @@ async function publishToRoom(roomName: string,
 
     if (libp2p == null) {
         status = {code: grpc.status.UNKNOWN, message: "Libp2p instance not configured"}
-    } else if (!subscriptions.has(roomName)) {
-        status = {code: grpc.status.INVALID_ARGUMENT, message: `Not subscribed to ${roomName}`}
     } else {
-        const room = subscriptions.get(roomName);
-        //TODO ADD FROM
-        await room?.broadcast(message);
+        const room = new Room(libp2p, roomName)
+        await room?.broadcast({ content: message});
```

Comment:
Why encapsulate the message as 'content'? note that message is already of the form { content, sender, receiver }